diff --git a/OS X development/Elephant.xcodeproj/xcshareddata/xcschemes/Elephant.xcscheme b/OS X development/Elephant.xcodeproj/xcshareddata/xcschemes/Elephant.xcscheme
index c5bbff5..06ceaf8 100644
--- a/OS X development/Elephant.xcodeproj/xcshareddata/xcschemes/Elephant.xcscheme	
+++ b/OS X development/Elephant.xcodeproj/xcshareddata/xcschemes/Elephant.xcscheme	
@@ -70,6 +70,28 @@
             ReferencedContainer = "container:Elephant.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "PGHOST"
+            value = "192.168.99.100"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "PGUSER"
+            value = "postgres"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "PGPORT"
+            value = "32768"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "PGDATABASE"
+            value = "elephant"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+      </EnvironmentVariables>
       <AdditionalOptions>
       </AdditionalOptions>
    </LaunchAction>
